src/Cli/dotnet/commands/dotnet-tool/install/ToolInstallCommandParser.cs

```diff
@@ -38,6 +38,11 @@ internal static class ToolInstallCommandParser
             ArgumentHelpName = LocalizableStrings.FrameworkOptionName
         };
 
+        public static readonly Option<bool> CreateManifestIfNeededOption = new Option<bool>("--create-manifest-if-needed", LocalizableStrings.CreateManifestIfNeededOptionDescription)
+        {
+            ArgumentHelpName = LocalizableStrings.CreateManifestIfNeededOptionName
+        };
```

Comment:
Per @baronfel's response, we don't need to specify the ArgumentHelpName here:

```suggestion
        public static readonly Option<bool> CreateManifestIfNeededOption = new Option<bool>("--create-manifest-if-needed", LocalizableStrings.CreateManifestIfNeededOptionDescription);
```